build.gradle

```diff
@@ -196,6 +196,7 @@ subprojects {
     "org.elasticsearch:elasticsearch-cli:${version}": ':server:cli',
     "org.elasticsearch:elasticsearch-core:${version}": ':libs:elasticsearch-core',
     "org.elasticsearch:elasticsearch-nio:${version}": ':libs:elasticsearch-nio',
+    "org.elasticsearch:elasticsearch-xcontent:${version}": ':libs:xcontent',
```

Comment:
Please make this `x-content`. This is consistent with established naming conventions. Sorry.